### DISCUSSION TITLE
Please tell me how to connect to Discord

### DISCUSSION THREAD

**Discussion by selly3822** [pkgxdev/pkgx#919] [Help] [answered]
help me

**Comment by DropHunterXYZ** [comment]
Go into Settings, Connection and then Connect the Github and Twitter (X) accounts

**Comment by arbiankuncoro-lab** [comment, marked as answer]
Simple regist your github account if you don't have any github account. Then open your discord, go to user profile/setting, click connection and then click on github, follow the link, done. You can do it to connect with another social media with the same path.

My discord id : x0rex0x
